types/presto.d.ts

```diff
@@ -1,205 +1,7334 @@
 /**
- * Taken from @see {@link https://demo.castlabs.com/#/docs?q=clpp.Player}
+ * These typings will shortly become available on NPM as part of `@castlabs/prestoplay`.
+ * When they are, delete this file!
```

Comment:
They are now, aren't they?

Reply:
True! Yeah, we've been hanging this PR out to dry for so long that we managed to publish a new version of PRESTOplay in the meantime :D